docs/source/app-dev/index.rst

```diff
@@ -0,0 +1,80 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Writing applications using the Ledger API
+#########################################
+
+.. toctree::
+   :hidden:
+
+   services
+   daml-lf-translation
+
+DAML contracts are stored on a ledger. In order to exercise choices on those contracts, create new ones, or read from the ledger, you need to use the **Ledger API**. (Every ledger that DAML can run on exposes this same API.) And to do anything sophisticated, you'll want to write an application. 
+
+Resources available to you
+**************************
+
+- **The Java bindings**: a library to help you write idiomatic applications using the Ledger API in Java.
+
+  :doc:`Read the documentation for the Java bindings </app-dev/bindings-java/index>`
+- **The experimental node.js bindings**: a library to help you write idiomatic applications using the Ledger API in JavaScript.
```

Comment:
```suggestion
- **The experimental Node.js bindings**: a library to help you write idiomatic applications using the Ledger API in JavaScript.
```

Reply:
Sorry for the nitpicking, unfortunately getting this right is very difficult as it's easy to find this spelled in very different ways. I'd suggest trying to follow this, which is how it's spelled out in the official material.